colossalai/fx/passes/meta_info_prop.py

```diff
@@ -236,3 +242,64 @@ def propagate(self, *args):
             Any: The value returned from executing the Module
         """
         return super().run(*args)
+
+    def summary(self, format: str = 'MB') -> str:
```

Comment:
should it be `unit` instead of `format`?

Reply:
yes